CHANGELOG.md

```diff
@@ -16,13 +16,14 @@
 
 - The methods `refreshTokenGrant`,`userTokenGrant` and `clientCredentialsGrant` of the `xsuaa-service` accept now a `string` or `XsuaaServiceCredentials` as first argument.
 This argument represents the `URL` used to fetch the access token from and represent two cases.
-A `string` values denotes the URL of the token service and it is not changed.
-A  `XsuaaServiceCredentials` denotes that the XSUAA is used to receive the token and a suffix `oauth/token` is appended to the URL property within `XsuaaServiceCredentials`.
+A `string` value denotes the URL of the token service where it is not changed.
+A `XsuaaServiceCredentials` denotes that the XSUAA service is used to receive the token and a suffix `oauth/token` is appended to the URL property within `XsuaaServiceCredentials`.
```

Comment:
In case it is a `string` it is treated as the URL to fetch the access token from.
In case it is an object of type `XsuaaServiceCredentials`, the the URL is built by appending `oauth/token` to the `.url` of the given object.